backend/src/main/java/gov/cdc/usds/simplereport/service/supportescalation/SlackWebhookService.java

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.usds.simplereport.service.supportescalation;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class SlackWebhookService {
+
+  private SlackConfigService slackConfigService;
+
+  public boolean sendSlackEscalationMessage() {
```

Comment:
Should this method have the `@AuthorizationConfiguration.RequireGlobalAdminUser` annotation?